stdlib/public/stubs/UnicodeNormalization.cpp

```diff
@@ -288,8 +288,8 @@ swift::__swift_stdlib_unorm2_getNFCInstance(__swift_stdlib_UErrorCode *err) {
 }
 
 int32_t swift::__swift_stdlib_unorm2_normalize(
-    const __swift_stdlib_UNormalizer2 *norm, const __swift_stdlib_UChar *src,
-    __swift_int32_t len, __swift_stdlib_UChar *dst, __swift_int32_t capacity,
+    const __swift_stdlib_UNormalizer2 *norm, const __swift_int32_t *src,
```

Comment:
You mean `uint16_t`? Why are we doing all this churn anyways?

Reply:
Good catch. I'm not sure why we started #if-ing the typedef for __swift_stdlib_UChar. Maybe the ICU headers differ on some other flavor of Linux?